--- linux-user/elfload.c.orig	2013-01-31 17:32:32.279066381 +0100
+++ linux-user/elfload.c	2013-01-31 17:32:46.815066380 +0100
@@ -1149,7 +1149,7 @@
     return (elf_check_arch(ehdr->e_machine)
             && ehdr->e_ehsize == sizeof(struct elfhdr)
             && ehdr->e_phentsize == sizeof(struct elf_phdr)
-            && ehdr->e_shentsize == sizeof(struct elf_shdr)
+            //&& ehdr->e_shentsize == sizeof(struct elf_shdr)
             && (ehdr->e_type == ET_EXEC || ehdr->e_type == ET_DYN));
 }
 
